sinks/sinks_service_test.go

```diff
@@ -168,15 +200,29 @@ func TestUpdateSink(t *testing.T) {
 			token: token,
 			err:   nil,
 		},
+		"update sink tags with new tags": {
+			sink:  newTagsSink,
+			tags:  newTags,
+			token: token,
+			err:   nil,
+		},
```

Comment:
I understand you are using variables like: newTags for multiple places, but for test cases, let's be more readable friendly instead of optimal, so we don' t need to go back and forth reading the test case.